device/arista/x86_64-arista_7060dx5_32/Arista-7060DX5-32-25Gx96-100Gx8-200Gx8/port_config.ini

```diff
@@ -0,0 +1,114 @@
+#name          lanes              alias        index  speed     fec
```

Comment:
Is this file required?

Reply:
which lane pattern is not supported?  4x25G?  would be good to know if there is a limitation in the design?

Reply:
The 4 sub-ports of the first FP port are using lane 1, 5, 3, 7
#name          lanes              alias        index  speed     fec
Ethernet0      1                  Ethernet1/1  1      25000
Ethernet1      5                  Ethernet1/2  1      25000
Ethernet2      3                  Ethernet1/3  1      25000
Ethernet3      7                  Ethernet1/4  1      25000

In platform.json, the first FP port is defined with lane 1, 2, 3, 4, 5, 6, 7, 8: https://github.com/sonic-net/sonic-buildimage/blob/master/device/arista/x86_64-arista_7060dx5_32/platform.json#L221

For the syntax of breakout_mode, we cannot define a new breakout_mode in platform.json to represent the pattern of 1, 5, 3, 7 for the 4 sub-ports

Reply:
Hmm,  the use case is to use QSFP DAC cable (not QSFPDD),  and the physical breakout port 1 map to Ethernet0, breakout port 2 map to Ethernet1 and so on.  Can you confirm if that is the mapping in your design?

Reply:
Yes, physical port 1, 2, 3, 4 on the breakout cable are mapped to Ethernet0, 1, 2, 3 in SONiC.

Reply:
OK, please test the HWSKU with QSFP DAC cable (not QSFPDD) for those 25G ports.   If results (port mapping and ports up status) good, we can close this comment.

Reply:
Yes, this has been tested on QSFP DAC (100GBASE-CR4)